src/styles/main.scss

```diff
@@ -1,3 +1,68 @@
+@import './variables';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  height: 100vh;
+}
+
+.calendar {
+  margin: 0 auto;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: ($cell-size + 1px) * 7;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  &__cell {
+    position: relative;
+    width: $cell-size;
+    height: $cell-size;
+    background-color: #eee;
+    border: 1px solid black;
+    padding: 10px;
+
+    @for $count from 1 through 31 {
+      &:nth-child(#{$count}n)::before {
+        content: "#{$count}";
+        width: 80%;
+        height: 80%;
+        position: absolute;
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
+        display: flex;
+        align-items: center;
+        justify-content: center;
```

Comment:
```suggestion

```
Consider adding these styles once to the needed class instead of generating them for each element in the cycle